report.md

```diff
@@ -29,36 +29,47 @@ The eight functions are presented in the table below. Every function has a CCN o
 
 | Function                                                            | Location                         | CCN, Lizard | CCN, manual (Student 1) | CCN, manual (Student 2) | NLOC, Lizard |
 | ------------------------------------------------------------------- | -------------------------------- | ----------- | ----------------------- | ----------------------- | ------------ |
-| Dijkstra(graph, V, src)                                             | graphs/dijkstra_2.py             | 8           | -                       | -                       | 15           |
+| dj_oracle(case, num_qubits)                                         | 9                                | 9           | -                       | 25                      |
```

Comment:
I found the same CCN, you can add 9 to "manual student 2"

Reply:
Yes, I'll add that too.